src/internal/components/InternalThemeEditor.tsx

```diff
@@ -45,6 +45,14 @@ export const InternalThemeEditor = ({
 }: InternalThemeEditorProps) => {
   const [canEdit, setCanEdit] = useState<boolean>(false); // helps sync puck preview and save state
 
+  const {
+    history: { setHistories },
+  } = usePuck();
```

Comment:
Does this work? I thought usePuck had to be inside a <Puck> component

Reply:
You're right, just moved it to ThemeHeader and it works correctly now. I had forgotten to re-pack my changes before testing it in InternalThemeHeader.